### ISSUE TITLE
matplotlib 3.2 throws error due to removed 'normed' argument

### ISSUE THREAD

**Issue by ialarmedalien** [Ecogenomics/CheckM#251]
Matplotlib v3.2, the current version installed by pip, causes an error to be thrown whenever `checkm` creates a histogram:

```
Unexpected error: <class 'AttributeError'>
Traceback (most recent call last):
  File "/miniconda/bin/checkm", line 611, in <module>
    checkmParser.parseOptions(args)
  File "/miniconda/lib/python3.6/site-packages/checkm/main.py", line 879, in parseOptions
    self.distributionPlots(options)
  File "/miniconda/lib/python3.6/site-packages/checkm/main.py", line 494, in distributionPlots
    plots.plot(f, tetraSigs, options.distributions)
  File "/miniconda/lib/python3.6/site-packages/checkm/plot/distributionPlots.py", line 47, in plot
    gcPlots.plotOnAxes(fastaFile, distributionsToPlot, axesHistGC, axesDeltaGC)
  File "/miniconda/lib/python3.6/site-packages/checkm/plot/gcPlots.py", line 89, in plotOnAxes
    axesHist.hist(data, bins=bins, normed=True, color=(0.5, 0.5, 0.5))
  File "/miniconda/lib/python3.6/site-packages/matplotlib/__init__.py", line 1565, in inner
    return func(ax, *map(sanitize_sequence, args), **kwargs)
  File "/miniconda/lib/python3.6/site-packages/matplotlib/axes/_axes.py", line 6808, in hist
    p.update(kwargs)
  File "/miniconda/lib/python3.6/site-packages/matplotlib/artist.py", line 1006, in update
    ret = [_update_property(self, k, v) for k, v in props.items()]
  File "/miniconda/lib/python3.6/site-packages/matplotlib/artist.py", line 1006, in <listcomp>
    ret = [_update_property(self, k, v) for k, v in props.items()]
  File "/miniconda/lib/python3.6/site-packages/matplotlib/artist.py", line 1002, in _update_property
    .format(type(self).__name__, k))
AttributeError: 'Rectangle' object has no property 'normed'
```

From [matplotlib API changes for 3.2.0](https://matplotlib.org/3.2.1/api/api_changes.html):

> ### Removals
> (...)
> * the `normed` kwarg to hist (use `density` instead)

i.e.
```
axesHist.hist(data, bins=bins, normed=True, color=(0.5, 0.5, 0.5))
```
=>
```
axesHist.hist(data, bins=bins, density=True, color=(0.5, 0.5, 0.5))
```

GitHub search reveals about five or six places in the codebase where this substitution should be made.

**Comment by dparks1134** [comment]
Hi. Thanks for bringing this to my attention. I have updated the code which will be incorporated in the next CheckM release.

**Comment by SithWijesinghe** [comment]
> Hi. Thanks for bringing this to my attention. I have updated the code which will be incorporated in the next CheckM release.

Hi @dparks1134 , would the pypi repository be updated with these changes soon?

**Comment by dparks1134** [comment]
Hi. I just released v1.1.3 which addresses this issue.
